components/camera/videosource/webcam.go

```diff
@@ -514,6 +514,22 @@ func (c *monitoredWebcam) Projector(ctx context.Context) (transform.Projector, e
 	return c.exposedProjector.Projector(ctx)
 }
 
+func (c *monitoredWebcam) Images(ctx context.Context) ([]image.Image, time.Time, error) {
+	if c, ok := c.underlyingSource.(camera.ImagesSource); ok {
+		return c.Images(ctx)
+	}
+	img, release, err := camera.ReadImage(ctx, c.underlyingSource)
+	if err != nil {
+		return nil, time.Time{}, errors.Wrap(err, "monitoredWebcam: call to get Images failed")
+	}
+	defer func() {
+		if release != nil {
+			release()
+		}
+	}()
+	return []image.Image{img}, time.Now(), nil
```

Comment:
I'm not sure how precise the customer wants this time to be but the [V4L2 buffer](https://www.kernel.org/doc/html/v4.9/media/uapi/v4l/buffer.html#struct-v4l2-buffer) already has timestamp we could return. Though, making changes to pass that up will be a lot more involved.

I'm guessing the reason `camera.ReadImage` is called here is to reduce the latency between the timestamp and function call. Is that right?

Reply:
More so I call ReadImage because that's the way I know how to get a single frame returned from the camera stream. 

I could make a ticket to mention what you say about using the V4L2 timestamp in the response

Reply:
Gotcha. Creating a ticket would be helpful!

Reply:
https://viam.atlassian.net/browse/RSDK-4062